docs/src/pages/index.tsx

```diff
@@ -1,37 +1,259 @@
+      </Heading>
+      <p>{props.secondaryText}</p>
+    </div>
+  );
+  if (props.inverted) {
+    return (
+      <div className="row margin-right--lg margin-left--lg" style={{ alignItems: 'center' }}>
+        {props.image}
+        {text}
+      </div>
+    );
+  } else {
+    return (
+      <div className="row margin-right--lg margin-left--lg" style={{ alignItems: 'center' }}>
+        {text}
+        {props.image}
+      </div>
+    );
+  }
+}
+
+function Features() {
+  const { colorMode } = useColorMode();
+  const imageTextRendering = (
+    <div className="row margin--lg">
+      <div>
+        <FaImage style={{ fontSize: 80, marginBottom: 30 }} />
+      </div>
+      <div>
+        <FaFont style={{ fontSize: 80, marginTop: 30 }} />
+      </div>
+    </div>
+  );
+  const wgpuLogo =
+    colorMode === 'dark' ? (
+      <WebGpuLogoDark className="margin--lg" width={100} height={100} />
+    ) : (
+      <WebGpuLogoLight className="margin--lg" width={100} height={100} />
+    );
+
+  return (
+    <div className="container margin-top--lg margin-bottom--md">
+      <Heading as="h2" className="margin-bottom--md">
+        Features
+      </Heading>
+
+      <Feature
+        text="WebGPU APIs"
+        secondaryText="Use existing WebGPU-based components provided by the compositor. If you need anything not covered by the existing components, you can always write your own WGSL shader that implements that effect."
+        image={wgpuLogo}
+      />
+      <Feature
+        text="Real-time processing"
+        secondaryText="Process live video sources in real-time with low latency."
+        image={<MdStream className="margin--lg" style={{ fontSize: 100 }} />}
+        inverted
+      />
+      <Feature
+        text="Static content"
+        secondaryText="Render text and static images onto your scene or pass them to other components for further processing. Currently we support: PNG, JPEG, SVG, and GIFs."
+        image={imageTextRendering}
+      />
+      <Feature
+        text="Web rendering"
+        inverted
+        secondaryText="You can render any website and combine it with video streams or other elements using the Chromium browser embedded inside the compositor."
+        image={<FaChrome className="margin--lg" style={{ fontSize: 100 }} />}
+      />
+    </div>
+  );
+}
+
+function IntegrationMembranePlugin() {
+  return (
+    <div className="row" style={{ alignItems: 'center' }}>
+      <MembraneLogo height={220} width={220} className="margin--lg" />
+      <div className="card col">
+        <div className="card__header">
+          <Heading as="h3" className="margin-top--sm">
+            Membrane plugin
+          </Heading>
+        </div>
+        <div className="card__body container">
+          <p>
+            Membrane is a developer-friendly multimedia framework for Elixir. You can easily add
+            video composing functionality into your multimedia pipeline using Live Compositor
+            plugin.
```

Comment:
It's a proper name, so use `Membrane Live Compositor Plugin` everywhere
```suggestion
            Membrane is a developer-friendly multimedia framework for Elixir. You can easily add
            video composing functionality into your multimedia pipeline using Live Compositor
            Plugin.
```